openwisp_monitoring/check/settings.py

```diff
@@ -3,7 +3,21 @@
 CHECK_CLASSES = getattr(
     settings,
     'OPENWISP_MONITORING_CHECK_CLASSES',
-    (('openwisp_monitoring.check.classes.Ping', 'Ping'),),
+    (
+        ('openwisp_monitoring.check.classes.Ping', 'Ping'),
+        ('openwisp_monitoring.check.classes.ConfigModified', 'Config Modified'),
+    ),
 )
 AUTO_PING = getattr(settings, 'OPENWISP_MONITORING_AUTO_PING', True)
+AUTO_CONFIG_MODIFIED = getattr(
+    settings, 'OPENWISP_MONITORING_AUTO_CONFIG_MODIFIED', True
+)
+# Input in minutes
+CONFIG_MODIFIED_MAX_TIME = getattr(
+    settings, 'OPENWISP_MONITORING_CONFIG_MODIFIED_MAXIMUM_TIME', 5
+)
+# Input in days
+CONFIG_MODIFIED_RETENTION_POLICY = getattr(
+    settings, 'OPENWISP_MONITORING_CONFIG_STATUS_RETENTION_POLICY', '48h0m0s'
```

Comment:
Please follow the standard we use for setting names: the user defines `OPENWISP_MONITORING_X_Y_Z` but we use `X_Y_Z` internally

Reply:
Can there be an exception here. I have named the setting as `OPENWISP_MONITORING_DEVICE_CONFIGURATION_X_Y_Z` but internally the settings are named as `CONFIG_X_Y_Z`. The reason was to explicitly say to newcomers that the setting is related to `DEVICE_CONFIGURATION` (there's also chart config and soon there shall be metric_config) but internally i.e. in the `check settings` we know that it means device config hence it's shortened. Let me know if it seems right?